metricbeat/module/logstash/node/_meta/fields.yml

```diff
@@ -4,10 +4,12 @@
     node
   release: ga
   fields:
+    # ECS?
     - name: host
       type: keyword
       description: >
         Host name
+    # Service.version?
     - name: version
```

Comment:
Based on testing this looks like the Logstash version, e.g. 7.0.0. So should this be changed to `service.version`?

Reply:
Yes 👍

Reply:
Changed to `service.version` in https://github.com/elastic/beats/pull/10350.